<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InVarWindowClass</class>
 <widget class="QDialog" name="InVarWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>InVarWindow</string>
  </property>
  <widget class="QRadioButton" name="radioButtonInVar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>内变量</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonForce">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>115</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>内力</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditInVar">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditForce">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>单元选择</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>变量设置</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxAll">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>全部单元</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxSel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>部分单元</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditElement">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>350</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSure">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>350</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
